lang/cs/Org.Apache.REEF.IMRU/OnREEF/IMRUTasks/TaskHostBase.cs

```diff
@@ -157,7 +157,7 @@ public void OnNext(ICloseEvent closeEvent)
         /// <summary>
         /// Dispose function. Dispose IGroupCommunicationsClient.
         /// </summary>
-        public void Dispose()
+        public virtual void Dispose()
```

Comment:
`IDisposable` is an interface, without implementation by definition, so any such keywords are redundant.
http://stackoverflow.com/questions/3621410/why-are-c-sharp-interface-methods-not-declared-abstract-or-virtual